sdk/ml/azure-ai-ml/tests/pipeline_job/e2etests/test_pipeline_job.py

```diff
@@ -1344,6 +1344,133 @@ def test_pipeline_node_with_default_component(self, client: MLClient, randstr: C
             == "microsoftsamples_command_component_basic@default"
         )
 
+    def test_register_output_without_name_cli(self, client: MLClient):
```

Comment:
These invalid case should goes to unittest as it does not triger any rest api call

Reply:
I move these tests to dsl/unittests/test_dsl_pipeline.py and pipeline_job/unittests/test_pipeline_job_validate.py